java/starter/src/Main.java

```diff
@@ -36,7 +37,7 @@ public RuntimeOutput main(RuntimeContext context) throws Exception {
         json.put("learn", "https://appwrite.io/docs");
```

Comment:
Lets replace motto with new motto after rebrand. You can see it in appwrite/templates starters